epoch_multinode_test.go

```diff
@@ -212,8 +211,14 @@ type testNode struct {
 		msg  *Message
 		from NodeID
 	}
-	l *testutil.TestLogger
-	t *testing.T
+	l  *testutil.TestLogger
+	t  *testing.T
+	bb *testControlledBlockBuilder
+}
+
+// triggerBlockShouldBeBuilt signals this nodes block builder it is expecting a block to be built.
+func (t *testNode) triggerBlockShouldBeBuilt() {
+	t.bb.blockShouldBeBuilt <- struct{}{}
```

Comment:
I think we should have a select here:

```
select {
    case  t.bb.blockShouldBeBuilt <- struct{}{}
    default:
```

Because if we ever move between rounds too quickly, we don't get to dispatch the monitor task, so we never need to read from this channel.